content/docs/next-release/troubleshooting.md

```diff
@@ -4,4 +4,68 @@ description: Solve commonly encountered issues
 weight: 10
 ---
 
-We are preparing a troubleshooting guide for you. In the meantime, make sure to check the blog post ["Help! Something is wrong with my Jaeger installation!"](https://medium.com/jaegertracing/help-something-is-wrong-with-my-jaeger-installation-68874395a7a6)
\ No newline at end of file
+Jaeger is composed of different components, each potentially running in its own host. It might be the case that one of these moving parts isn’t working properly, causing spans to not be processed and stored. When something goes wrong, make sure to check the items listed here.
+
+## Sampling strategy
+
+Before everything else, make sure to confirm what is the sampling strategy. By default, Jaeger uses a probabilistic sampling strategy, with a 1/1000th chance that the span will be reported.
+
+Typically, the sampling strategy can be set via the environment variables `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, but refer to the Jaeger Client’s documentation for the language you are using for more details about which sampling strategies are available. When using the Jaeger Java Client, the strategy is usually printed out via the logging facility provided by the instrumented application when creating the tracer:
+
+```
+2018-12-10 16:41:25 INFO  Configuration:236 - Initialized  tracer=JaegerTracer(..., sampler=ConstSampler(decision=true,  tags={sampler.type=const, sampler.param=true}), ...)
+```
+
+## Logging reporter
+
+Some Jaeger clients are able to log the spans that are being reported to the logging facility provided by the instrumented application. Typically, this can be done by setting the environment variable `JAEGER_REPORTER_LOG_SPANS` to `true`, but refer to the Jaeger Client’s documentation for the language you are using. When using the Jaeger _Java_ Client, spans are reported like the following:
+
+    2018-12-10 17:20:54 INFO  LoggingReporter:43 - Span reported:  e66dc77b8a1e813b:6b39b9c18f8ef082:a56f41e38ca449a4:1 -  getAccountFromCache
+
+The log entry above contains three IDs: the trace ID `e66dc77b8a1e813b`, the span’s ID `6b39b9c18f8ef082` and the span’s parent ID `a56f41e38ca449a4`. When the backend components have the log level set to `debug`, the span and trace IDs should be visible on their standard output (see more about that below, under “Increase the logging in the backend components”).
+
+The logging reporter follows the sampling decision made by the sampler, meaning that if the span is logged, it should also reach the the agent or collector.
```

Comment:
"the the"

Reply:
Good catch! It was a problem in the original blog post as well.